pkg/pillar/hypervisor/kvm.go

```diff
@@ -334,11 +334,19 @@ const qemuPciPassthruTemplate = `
 `
 const qemuSerialTemplate = `
 [chardev "charserial-usr{{.ID}}"]
+{{- if ne .Machine "virt"}}
   backend = "tty"
+{{- else}}
+  backend = "serial"
+{{- end}}
```

Comment:
What about having it without double negation?
```suggestion
  backend = "tty"
  {{- if eq .Machine "virt"}}
    backend = "serial"
  {{- end}}
```

Reply:
do you mean like this?
```go
{{- if eq .Machine "virt"}}
  backend = "serial"
{{- else}}
  backend = "tty"
{{- end}}
```

Reply:
That's also ok.

Reply:
@rene are you swapping the code per the above?

Reply:
Yes @eriknordmark , it's done.